src/scvmm/azext_scvmm/commands.py

```diff
@@ -0,0 +1,137 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+
+from azure.cli.core import AzCommandsLoader
+
+from ._client_factory import (
+    cf_vmmserver,
+    cf_cloud,
+    cf_virtual_network,
+    cf_virtual_machine_template,
+    cf_virtual_machine,
+    cf_availability_sets,
+    cf_inventory_items,
+)
+
+from ._validators import (
+    validate_param_combos,
+    validate_param_combos_for_vm,
+    validate_param_combos_for_avset,
+)
+
+
+def load_command_table(self: AzCommandsLoader, _):
+
+    with self.command_group('scvmm vmmserver', client_factory=cf_vmmserver) as g:
+        g.custom_command('connect', 'connect_vmmserver', supports_no_wait=True)
+        g.custom_command('update', 'update_vmmserver', supports_no_wait=True)
+        g.custom_command('delete', 'delete_vmmserver', supports_no_wait=True)
```

Comment:
Please add "wait" commands for this command group. For example:
https://github.com/Azure/azure-cli-extensions/blob/aa0d22ef8b1283de7cdec0fd9374c7153d7d1ec5/src/front-door/azext_front_door/commands.py#L78

Reply:
Have added wait command everywhere. I had to change the `scvmm vm-template wait` command to `custom_wait_command`, because `--virtual-machine-template-name` parameter was exceeding the max length of parameter. Any suggestions if we could rename the parameter in the wait command without using custom wait?

Reply:
Hi @nascarsayan, you can suppress the max length of parameter linter validation by adding a linter_exclusions.yml file. For example: https://github.com/Azure/azure-cli-extensions/blob/6bb827feddd0e6c6913cbd553f32746a33acea7e/src/datafactory/linter_exclusions.yml#L1-L5

Reply:
Hi @kairu-ms, I've used `options_list` to use a shorter name for the param. Thanks for the references.

One question: We don't have a `wait` on a particular disk/nic of the VM, is that fine? The wait we are providing for `scvmm vm disk` / `scvmm vm nic` are the same as `scvmm vm`.
These command groups use `VirtualMachinesOperations` under the hood, i.e., there's no separate `client_factory` for them.

Reply:
I see, that’s ok